python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1262,18 +1396,24 @@ def atspectrograph_ss_callback(self, data):
         """
 
         self.log.debug('Got new atspectrograph summary state, resetting filter/disperser offsets')
-        self.atspectrograph_summary_state = State(data.summaryState)
+        self.atspectrographSummaryState = State(data.summaryState)
         # remove offsets from previous spectrograph setup
-        self.focus_offset_yet_to_be_applied = -self.focus_offset_per_category['filter']
-        self.focus_offset_per_category['filter'] = 0.0
+        self.focusOffsetYetToBeApplied += -self.focusOffsetPerCategory['filter']
+        self.focusOffsetPerCategory['filter'] = 0.0
+
+        self.focusOffsetYetToBeApplied += -self.focusOffsetPerCategory['disperser']
+        self.focusOffsetPerCategory['disperser'] = 0.0
+
+        self.pointingOffsetsYetToBeApplied += -self.pointingOffsetsPerCategory['filter']
+        self.pointingOffsetsPerCategory['filter'] = np.array([0.0, 0.0])
 
-        self.focus_offset_yet_to_be_applied = -self.focus_offset_per_category['disperser']
-        self.focus_offset_per_category['disperser'] = 0.0
+        self.pointingOffsetsYetToBeApplied += -self.pointingOffsetsPerCategory['disperser']
+        self.pointingOffsetsPerCategory['disperser'] = np.array([0.0, 0.0])
 
     async def check_atspectrograph(self):
         """ Check that the atspectrograph is online and enabled"""
-        if self.atspectrograph_summary_state != State.ENABLED:
-            raise RuntimeError(f"ATSpectrograph (LATISS) in {self.atspectrograph_summary_state}. "
+        if self.atspectrographSummaryState != State.ENABLED:
+            raise RuntimeError(f"ATSpectrograph (LATISS) in {self.atspectrographSummaryState}. "
```

Comment:
Suggestion for the string formatting: 

```
f"ATSpectrograph (LATISS) in {State(self.atspectrographSummaryState)!r}.
f"Expected {State.ENABLED!r}. Enable CSC before "
f"activating corrections.")
```

Reply:
so i implemented this, then hacked a test to try it to see if it would actually work (just removing the event that publishes the spectrograph state), but it fails because "None" is not a state. 
I think there will be times where it will get a value of "None" even when deployed. So i ended up not implementing it.
![image](https://user-images.githubusercontent.com/14132079/83310430-d9e9ea80-a1c0-11ea-9db5-2ee4916adb8c.png)

Reply:
I lied, i partially implemented it.
![image](https://user-images.githubusercontent.com/14132079/83310572-5977b980-a1c1-11ea-8be3-618fdf0d52b2.png)